eZ/Publish/Core/MVC/Symfony/EventListener/ContentViewTwigVariablesSubscriber.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Publish\Core\MVC\Symfony\EventListener;
+
+use eZ\Publish\Core\MVC\ConfigResolverInterface;
+use eZ\Publish\Core\MVC\Symfony\ExpressionLanguage\ExpressionLanguage;
+use eZ\Publish\Core\MVC\Symfony\ExpressionLanguage\TwigVariableProviderExtension;
+use eZ\Publish\Core\MVC\Symfony\Event\PreContentViewEvent;
+use eZ\Publish\Core\MVC\Symfony\MVCEvents;
+use eZ\Publish\Core\MVC\Symfony\View\VariableProviderRegistry;
+use eZ\Publish\Core\MVC\Symfony\View\View;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+final class ContentViewTwigVariablesSubscriber implements EventSubscriberInterface
+{
+    private const EXPRESSION_INDICATOR = '@=';
+
+    public const PARAMETERS_KEY = 'params';
+
+    /** @var \eZ\Publish\Core\MVC\Symfony\View\VariableProviderRegistry */
+    private $parameterProviderRegistry;
+
+    /** @var \eZ\Publish\Core\MVC\ConfigResolverInterface */
+    private $configResolver;
+
+    /** @var \Symfony\Component\ExpressionLanguage\ExpressionLanguage */
+    private $expressionLanguage;
+
+    public function __construct(
+        VariableProviderRegistry $parameterProviderRegistry,
+        ConfigResolverInterface $configResolver
+    ) {
+        $this->parameterProviderRegistry = $parameterProviderRegistry;
+        $this->configResolver = $configResolver;
+        $this->expressionLanguage = new ExpressionLanguage();
```

Comment:
What about https://github.com/ezsystems/ezplatform-core/pull/14#discussion_r375798235 ?

Reply:
Let's keep it as it is for know. We'll think about extensibility point and our abstraction over Symfony later as a separate feature.